CONTRIBUTING.md

```diff
@@ -103,3 +102,59 @@ developers.
 
   More information about tool development can be found [on the
   wiki](https://wiki.galaxyproject.org/Develop).
+
+## Handling Pull Requests
+
+Everyone is encouraged to express opinions and issue non-binding votes on pull
+requests, but only members of the *committers* group may issue binding votes
+on pull requests. Information on the *committers* group can be found in the
+[organization document](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst)
+describing governance of the core Galaxy code base.
+
+Votes should on pull requests should take the form of
+[+1, 0, -1, and fractions](http://www.apache.org/foundation/voting.html)
+as outlined by the Apache Foundation.
+
+Pull requests modifying pre-existing releases should be restricted to bug fixes
+and require at least 2 *+1* binding votes from someone other than the author of
+the pull request with no *-1* binding votes.
+
+Any pull request changing or clarifying the procedures governing this
+repository must remain open for at least 4 days and require at least 3 *+1*
+votes with no *-1* votes from qualified *committers*.
+
+Any other pull request requires at least 1 binding *+1* vote from someone other
+than the author of the pull request.
+
+### Timelines
+
+Except in the case of pull requests modifying governance procedures, there are
+generally no objective guidelines defining how long pull requests must remain
+open for comment. Subjectively speaking though - larger and more potentially
+controversial pull requests containing enhancements should remain open for a at
+least a few days to give everyone the opportunity to weigh in.
+
+### Vetos
+
+A note on vetos (*-1* votes) taken verbatim from the
+[Apache Foundation](http://www.apache.org/foundation/voting.html):
+
+>"A code-modification proposal may be stopped dead in its tracks by a -1 vote
+by a qualified voter. This constitutes a veto, and it cannot be overruled nor
+overridden by anyone. Vetos stand until and unless withdrawn by their casters.
+>
+>To prevent vetos from being used capriciously, they must be accompanied by a
+technical justification showing why the change is bad (opens a security
+exposure, negatively affects performance, etc. ). A veto without a
+justification is invalid and has no weight."
+
+### Direct Commit Access
+
+The Galaxy *committers* group may only commit directly to Galaxy (i.e.  outside
+of a pull request and not following the procedures described here) the
+following two categories of patches:
+
+* Serious security vulnerabilities must simultaneously be made to multiple
```

Comment:
Maybe "Serious security vulnerability patches, which must" (or some variant which clearly delineates the stipulation clause on "srs sec vulns")